samples/snippets/csharp/VS_Snippets_CLR/IO.FileSystem.AccessControl/DirectoryInfoCreateDirectory.cs

```diff
@@ -0,0 +1,35 @@
+using System.IO;
+using System.Runtime.Versioning;
+using System.Security.AccessControl;
+using System.Security.Principal;
+
+namespace MyNamespace
+{
+    public class MyClassCS
+    {
+        // Attribute to address CA1416 warning:
+        // System.IO.FileSystem.AccessControl APIs are  only available on Windows
+        [SupportedOSPlatform("windows")]
+        static void Main()
+        {
+            // Create the file security object
+
+            SecurityIdentifier identity = new SecurityIdentifier(WellKnownSidType.BuiltinUsersSid, null);
```

Comment:
I would add a comment explaining what `BuiltinUsersSid` is and why the 2nd parameter can be null. 

```suggestion
            SecurityIdentifier identity = new SecurityIdentifier(
                WellKnownSidType.BuiltinUsersSid, // this maps to "Everyone" user group in Windows
                null); // null is OK for this particualr user group, for others a non empty value might be required
```

Reply:
Sounds good. I'll add it. I adjusted a small typo.